pkg/cloud/gcp/bigqueryintegration_types.go

```diff
@@ -113,6 +113,25 @@ func (ccl *CloudCostLoader) Load(values []bigquery.Value, schema bigquery.Schema
 			}
 
 			properties.ProviderID = ParseProviderID(resource)
+		case ResourceGlobalNameColumnName:
+			// skip if we already got ProviderID from resource.name, as resource.global_name is a fallback for when
+			// resource.name is null
+			if len(properties.ProviderID) > 0 {
+				continue
+			}
+
+			resourceGlobalNameValue := values[i]
+			if resourceGlobalNameValue == nil {
+				continue
```

Comment:
I think we would want to add `properties.ProviderID = ""` here before continuing right? Just in case we hit the case where all `resource.global_name`'s are nil

Reply:
Good catch! You're absolutely right.